ertviz/controllers/multi_response_controller.py

```diff
@@ -49,19 +49,26 @@ def _get_observation_plots(observation_df, x_axis):
     data = observation_df["values"]
     stds = observation_df["std"]
     x_axis = observation_df["x_axis"]
+    active_mask = observation_df["active"]
+
+    style = assets.ERTSTYLE["response-plot"]["observation"]
+    color = [style["color"] if active else "rgb(0, 0, 0)" for active in active_mask]
```

Comment:
shall we use other color (some lighter tone of red) than black?

Reply:
We can - i tried lighter red but i didn't get it to "stand out" as a deactivated one.... tried grey as well - but it is kind of difficult.... you have any suggestions?

Reply:
Did some experiments: `rgb(255,127,0)` looks pretty decent.
![image](https://user-images.githubusercontent.com/6409506/110936887-d8457980-8331-11eb-812f-75db1181475f.png)

Reply:
agree - stand out at least! But - does it communicate `inactive` or `deactivated`?  🤔  From the look of it i cannot interpret if it is an active obs or not. With black / grey / faded / more transparent my brain immediately thinks something is turned off...